tests/nebula-test-run.py

```diff
@@ -110,7 +110,7 @@ def run_tests(self, args):
         # Switch to your $src_dir/tests
         os.chdir(TEST_DIR)
         error_code = executor.run_tests(args)
-        # data_loader.drop_data()
+        data_loader.drop_data()
```

Comment:
@laura-ding Is it necessary?

Reply:
necessary, maybe someone annotation it before.